src/modules/migration/MigrationListItem.tsx

```diff
@@ -101,7 +101,12 @@ export const MigrationListItem = ({
       />
     );
   return (
-    <ListItem sx={{ flexDirection: 'column', pl: 0 }}>
+    <ListItem
+      sx={{ flexDirection: 'column', pl: 0 }}
+      data-cy={`migration-${borrowApyType != undefined ? 'borrow' : 'supply'}-${
+        userReserve.reserve.symbol
+      }${borrowApyType != undefined ? '-' + borrowApyType : ''}`}
+    >
```

Comment:
Although this works here in this case, `borrowApyType != undefined`, it'd probably be better to use strict equality check (double equals):

 `borrowApyType !== undefined`

Reply:
fixed